utils/fmoconfig.py

```diff
@@ -156,6 +161,25 @@ def set_vmddp_config(self, vmname: str, newconf: Dict):
         for k in newconf.keys():
             config[k] = newconf[k]
 
+    def write_vmddp_config(self) -> None:
+        vmlist = self.get_vms_names_list()
+        vmddplist = []
+        for vmname in vmlist:
+            dpconfig = self._get_vmddp_config(vmname).get_config()
+            try:
+                if (dpconfig.get("enable",False)):
+                    devicelist = dpconfig.get("devices", [])
+                    for device in devicelist:
+                        device["vendorId"]=device.pop("vendorid")
+                        device["productId"]=device.pop("productid")
+                    vmddplist.append({'name': vmname, 'qmpSocket': '/var/lib/microvms/'+vmname+'/'+vmname+'.sock', 'usbPassthrough': devicelist})
+            except Exception as e:
+                continue
+        vhotplugconf = {"vms": vmddplist }
+        os.makedirs(os.path.dirname(self.__ddp_path_wo), exist_ok=True)
```

Comment:
Everything looks good, except this. I assume that user may also want to manually modify the config. So, please have a look here: [link](https://github.com/tiiuae/FMO-OS/blob/26c20046a41dc872d565636d682bc4701d1ad18b/modules/dynamic-portforwarding-service-host/default.nix#L9)
FMO-tool should be used in service every time it's restarted. And generate config there

Reply:
Because DDP uses a separate JSON config file, so I believe there should be `__ddp_path_wo`.
And that file is generated from our YAML config file and only based on it. It should only be an internal use for the script.

So what I think we should do is to _hide_ the JSON config, or not allow to manually modify it. If there are conflicts/differences between our YAML and JSON config, it will cause quite alot confusion of which DDP config should be taken into account. 
In the case of manually modifying the YAML config in DDP sections, users can `[fmo-tool ddp generate](https://github.com/tiiuae/FMO-OS/blob/26c20046a41dc872d565636d682bc4701d1ad18b/modules/dynamic-device-passthrough-services-host/default.nix#L23)` to recreate the JSON config based on YAML, but personally I wouldn't suggest that

Reply:
Yep, DPF also has dedicated config. But, this tool supposed just to generate config like: `fmo-tool ddp rules -r > /path/to/json.config`, but not handle this config as is

Reply:
okay, I thought a little bit more: that is fine, lets keep this as is. I will rework DPF